reason-react-native/src/apis/StyleSheet.md

```diff
@@ -1,20 +1,104 @@
 ---
 id: apis/StyleSheet
 title: StyleSheet
-wip: true
+officialDoc: https://facebook.github.io/react-native/docs/stylesheet
 ---
 
+`StyleSheet` mainly has a [method](#create) to create an object containing many
+styles, similar to a CSS StyleSheet. This is helpful to keep style definitions
+at fewer locations, away from the render function. Separating concerns in your
+code in this manner should help readability. Named styles might also help
+differentiate between your components. Finally, allowing components to refer to
+styles by ID, it also reduces the amount of data transferred over the JS bridge.
+
+There is also a [method](#flatten) to merge multiple styles into a single style
+and several [constants](#constants) for commonly used style definitions.
+
+## Methods
+
+### `create`
+
+Takes and returns `Js.t('a)` objects, with `string` keys and `Style.t` values.
+Individual styles can be accessed using `##`, followed by the name specified as
+the key.
+
+```reason
+create: Js.t('a) => Js.t('a)
+```
```

Comment:
Same question as https://github.com/reasonml-community/bs-react-native/pull/519#discussion_r289291691

Reply:
Perhaps we revisit this issue later, but I don't think there is a way to exclude `array(Style.t)` without a lot of pain elsewhere.